app/views/referee_interface/reference/safeguarding.html.erb

```diff
@@ -12,7 +12,7 @@
       <%= f.govuk_error_summary %>
 
       <%= f.govuk_radio_buttons_fieldset :any_safeguarding_concerns, legend: { text: t('page_titles.referee.safeguarding', full_name: @application.full_name), size: 'l', tag: 'h1' } do %>
-        <%= f.govuk_radio_button :any_safeguarding_concerns, :yes, label: { text: t('referee.any_safeguarding_concerns.yes.label') } do %>
+        <%= f.govuk_radio_button :any_safeguarding_concerns, :yes, label: { text: t('referee.any_safeguarding_concerns.yes.label') }, link_errors: true do %>
```

Comment:
Thank you! The links not working in the error summary is a pet peeve of mine, as you know :)